python/mlx/nn/layers/activations.py

```diff
@@ -23,6 +23,47 @@ def relu(x):
     return mx.maximum(x, 0)
 
 
+def leaky_relu(x, alpha=0.01):
+    """Applies the Leaky Rectified Linear Unit.
+
+    Simply ``mx.maximum(alpha * x, x)``.
+    """
+    return mx.maximum(alpha * x, x)
+
+
+def elu(x, alpha=1.0):
+    """Applies the Exponential Linear Unit.
+
+    Simply ``mx.maximum(alpha * (mx.exp(x) - 1), x)``.
+    """
+    return mx.maximum(alpha * (mx.exp(x) - 1), x)
+
+
+def relu6(x):
+    """Applies the Rectified Linear Unit 6.
+
+    Applies :math:`\min(\max(x, 0), 6)` element wise.
+    """
```

Comment:
Docstrings with backslashes often need to be *raw* string literals meaning start the docstring with an r as you did for `log_sigmoid`.